### PR TITLE
Create footer for regulatory links

### PR DESCRIPTION
**Pull request by james-geiger**
Per #854 and feedback from @ApertureDevelopment, provides a mechanism to add links to a privacy policy and terms of service from the settings page.

Currently would not display on the page editor pages, but am open to and welcome feedback.  It just didn't seem to me to fit the flow very well with the full-page editor, but this could also be added as a setting as well.

@ssddanbrown, not sure what your thoughts are on limits to the number of "app settings" but this seemed like it would be a good fit for that section.  Also purposely limited this to two different kinds of links.  Could likely be abstracted into a feature where you could add any number of "custom footer links" each with a label and href, but this seemed like a good place to start.

### CODE REVIEW

File: resources/views/common/footer.blade.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@if(setting('app-privacy-policy') | setting('app-terms-of-service'))
```
Comment (Findus23) [inline resources/views/common/footer.blade.php:1]:
```suggestion
@if(setting('app-privacy-policy') or setting('app-terms-of-service'))
```
I am not too familiar with blade templates, but without this change, I only get a lot of errors (I only entered a privacy policy link, but not a terms of service)

Comment (james-geiger) [reply resources/views/common/footer.blade.php:1]: What version of PHP are you running, out of curiosity?

Comment (Findus23) [reply resources/views/common/footer.blade.php:1]: PHP 7.3

Comment (Findus23) [reply resources/views/common/footer.blade.php:1]: Thinking about it, I guess || would also be possible, but doesn't | do a bitwise or?

**Comment by ssddanbrown** [comment]
Thank you for this PR @james-geiger and sorry for my late response.

This all looked great, but I knew that people would probably want to change the wording and links so I built upon your work and made further changes in 44c41e9e4dbe31f071b8c87ffae77ad6737c2cbc to make this a configurable, order-able list. 

Will be part of the next feature release.

**Pull request merged**